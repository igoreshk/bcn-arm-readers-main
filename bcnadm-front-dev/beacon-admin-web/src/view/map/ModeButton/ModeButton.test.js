import React from 'react';
import { shallow } from 'enzyme';

import { ModeButton } from './index';

const props = {
  icon: <div />,
  link: 'link',
  onClick: jest.fn(),
  buttonId: 'buttonId',
  tooltip: 'tooltip',
  buttonType: 'buttonType',
  active: true
};

const wrapper = shallow(<ModeButton {...props} />);

describe('mock', () => {
  it('render component', () => expect(wrapper.length).toEqual(1));
  it('snapshot for ModeButton', () => expect(wrapper).toMatchSnapshot());
});
