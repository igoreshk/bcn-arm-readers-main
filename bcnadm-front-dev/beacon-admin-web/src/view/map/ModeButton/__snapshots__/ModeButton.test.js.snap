// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mock snapshot for ModeButton 1`] = `
<Fragment>
  <WithStyles(ForwardRef(IconButton))
    data-for="buttonId"
    data-tip={true}
    onClick={[MockFunction]}
    style={
      Object {
        "backgroundColor": "#39C2D7",
        "borderRadius": "3px",
        "boxShadow": "1px 2px 2px #999999",
        "height": "36px",
        "margin": "1px",
        "opacity": "1",
        "padding": "0px",
        "width": "36px",
      }
    }
  >
    <div />
  </WithStyles(ForwardRef(IconButton))>
  <ReactTooltip
    clickable={false}
    effect="solid"
    id="buttonId"
    insecure={true}
    place="bottom"
    resizeHide={true}
    type="light"
    wrapper="div"
  >
    <span
      style={
        Object {
          "fontSize": "12px",
          "fontWeight": "500",
          "lineHeight": "2.3",
        }
      }
    >
      tooltip
    </span>
  </ReactTooltip>
</Fragment>
`;
